src/main/java/com/y/java_board/controller/ArticleController.java

```diff
@@ -0,0 +1,104 @@
+package com.y.java_board.controller;
+
+import com.y.java_board.domain.Article;
+import com.y.java_board.domain.Comment;
+import com.y.java_board.dto.ArticleDto;
+import com.y.java_board.dto.CommentDto;
+import com.y.java_board.service.ArticleService;
+import com.y.java_board.service.CommentService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+
+@Controller()
+public class ArticleController {
+
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+    private final ArticleService articleService;
+    private final CommentService commentService;
+
+    public ArticleController(ArticleService articleService, CommentService commentService){
+        this.articleService = articleService;
+        this.commentService = commentService;
+    }
+
+    @GetMapping("/articles")
+    public String articles(Model model){
+        List<Article> articles = articleService.findArticles();
+        model.addAttribute("articles", articles);
+        return "article/articles";
+    }
+
+    @GetMapping("/articles/new")
```

Comment:
Do not use words like new in API. You can use the below:
1. POST -> /articles (body to insert the data)
2. GET all -> /articles
3. GET one article -> /articles/{article-id}

Like this.